### PR TITLE
Disable collect_list and collect_set for window by default

### PR DESCRIPTION
**Pull request by revans2**
This fixes https://github.com/NVIDIA/spark-rapids/issues/10110

### CODE REVIEW

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
The link checker is having problems with the release in progress.

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala [hunk 1]
```diff
@@ -726,6 +726,22 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
       .booleanConf
       .createWithDefault(true)
 
+  val ENABLE_WINDOW_COLLECT_LIST = conf("spark.rapids.sql.window.collectList.enabled")
+      .doc("The output size of collect list for a window operation is proportional to " +
+          "the window size squared. The current GPU implementation does not handle this well " +
+          "and is disabled by default. If you know that your window size is very small you " +
+          "can try to enable it")
```
Comment (jlowe) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:733]:
```suggestion
          "can try to enable it.")
```

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala [hunk 2]
```diff
@@ -726,6 +726,22 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
...
+      .booleanConf
+      .createWithDefault(false)
+
+  val ENABLE_WINDOW_COLLECT_SET = conf("spark.rapids.sql.window.collectSet.enabled")
+      .doc("The output size of collect set for a window operation can be proportional to " +
+          "the window size squared. The current GPU implementation does not handle this well " +
+          "and is disabled by default. If you know that your window size is very small you " +
+          "can try to enable it")
```
Comment (jlowe) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala:741]:
```suggestion
          "can try to enable it.")
```

**Review by jlowe — DISMISSED** [review]

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
@jlowe please take another look

**Review by jlowe — APPROVED** [review]

**Pull request merged**